docs/user_guide/assessment.rst

```diff
@@ -470,6 +470,19 @@ There are many other customizations that can be done. More information can be fo
 
 .. _dashboard:
 
+
+Plotting model comparisons
```

Comment:
>     * [ ]  More demonstration???

According to the requirements of the project such a change should have a proper docstring and an entry in the user guide. As you've already figured out it's sometimes easiest to add a file (or section to an existing file) and `literalinclude` it in the user guide with some more description and context.

Reply:
> According to the requirements of the project such a change should have a proper docstring and an entry in the user guide.

Sorry, I'm not quite sure which requirements. Is this from the contributor guide? And do you mean that I should add an example in the docstring of the function definition?

Reply:
Not sure how I missed this comment before! Sorry for the massive delay 🙁 
The [contrib guide](https://fairlearn.org/v0.7.0/contributor_guide/development_process.html#development-process) says 
> Documentation should be provided with pull requests that add or change functionality. This includes comments in the code itself, docstrings, and user guides.
So with doctoring and user guide you're good. However, since this is a plotting API the output won't show up in the user guide with a normal code block. Instead, you'd need to create an "example" in a python file and then use `literalinclude` to reference that code and it's outputs. We've done that in https://github.com/fairlearn/fairlearn/blob/main/docs/user_guide/assessment.rst under "Plotting grouped metrics" based on `plot_quickstart.py` from the examples and then included the `figure` which is the plot.

I don't actually know if this can be done in the API reference as well. I guess it can, although I wouldn't expect you to experiment with that in this PR. An example in the user guide explaining what this kind of plot is useful for is plenty. Does that make sense?